ros2_batch_job/__main__.py

```diff
@@ -384,18 +384,27 @@ def build_and_test(args, job):
     # xunit2 format is needed to make Jenkins xunit plugin 2.x happy
     with open('pytest.ini', 'w') as ini_file:
         ini_file.write('[pytest]\njunit_family=xunit2')
-    # check if packages have a pytest.ini file and add the xunit2
-    # format if it is not present
+    # check if packages have a pytest.ini file that doesn't choose junit_family=xunit2
+    # and patch configuration if needed to force the xunit2 value
+    xunit_6_or_greater = job.run([
+        '"%s"' % job.python, '-c', '\''
+        'from distutils.version import StrictVersion;'
+        'import pytest;'
+        'import sys;'
+        'sys.exit(StrictVersion(pytest.__version__) >= StrictVersion("6.0.0"))'
+        '\''])
     for path in Path('.').rglob('pytest.ini'):
         config = configparser.ConfigParser()
         config.read(str(path))
-        try:
-            # only if xunit2 is set continue the loop with the file unpatched
-            if config.get('pytest', 'junit_family') == 'xunit2':
+        if xunit_6_or_greater:
```

Comment:
Fix variable name to match earlier suggestion.

```suggestion
        if pytest_6_or_greater:
```